### ISSUE TITLE
Perplexity is calling the wrong model

### ISSUE THREAD

**Issue by JJsilvera1** [enricoros/big-AGI#642]
hello. When I use perplexity the call is sending to the wrong model type.

[Service Issue] Perplexity: Bad Request (400): "error": { "message": "Invalid model 'llama-3-sonar-large-32k-chat'. Permitted models can be found in the documentation at https://docs.perplexity.ai/docs/model-cards.", "type": "invalid_model", "code": 400 }

should be 3.1 models.

I just added api access today

**Comment by enricoros** [comment]
Thanks @JJsilvera1, fixed in Big-AGI-2.

![image](https://github.com/user-attachments/assets/f4edc539-00e0-4620-b091-b8db5ba2fe97)

**Comment by JJsilvera1** [comment]
Okay, thank you!

**Comment by JJsilvera1** [comment]
@enricoros I signed up for the beta but since I don't have access to it doesn't really help 😂

**Comment by enricoros** [comment]
Thanks for signing up, I'll personally review your answers.
![image](https://github.com/user-attachments/assets/176b9646-d424-4355-bc54-5f9e895af574)

**Comment by enricoros** [comment]
@JJsilvera1 - I can have a pre-pilot program and admit you to the 2 alpha server.

V2 is incredibly better than V1, multimodal, resilient, faster, shows token usages, pricings, supports new architectures, and a few new large features I'm not mentioning yet. etc. But some features are still not ported to V2, so it's taking me time and I want the top quality.

I've marked your email for pre-access, please follow up with me anytime.

**Comment by JJsilvera1** [comment]
Hello, I just signed up for the beta just now

**Comment by JJsilvera1** [comment]
@enricoros hello, I actually signed up but also I wanted to actually make a feature request that maybe most people don't recommend. 

I just want to be able to have more than one window of big AGI open. It would be cool because then I can run multiple prompts and cross reference them in different tabs, but it seems like I get an error anytime I duplicate the tab that one sessions are already open

**Comment by enricoros** [comment]
I agree it would be great. It's in the backlog of features, and we'll get to it once we scale the operation. This will require a centralized system / server.

**Comment by JJsilvera1** [comment]
Hi @enricoros can I get access to v2 please?

**Comment by gershte** [comment]
Hi @enricoros can I get access to v2 as well please?

**Comment by enricoros** [comment]
Closing as shipped.

The Perplexity list was rewritten for the 2 line and tracks the sonar family, so the dead llama-3-sonar ids are gone - and it stays current. On the other thing you asked in this thread, running several big-AGI windows at once: that is still not there, one window at a time is enforced deliberately, though a second tab can now take the session over instead of being turned away. It stays tracked in #670. Closing this one.

@JJsilvera1 this close comes from an automated backlog pass - if you disagree, feel free to reopen.
